api/src/pcapi/core/finance/factories.py

```diff
@@ -116,6 +117,10 @@ class PricingFactory(_BasePricingFactory):
     revenue = factory.LazyAttribute(lambda pricing: int(100 * pricing.booking.total_amount))
 
 
+class InvoicedPricingFactory(PricingFactory):
+    status = models.PricingStatus.INVOICED
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏Est-ce qu'on ferait pas mieux de faire un `PricingFactory(status=models.PrincingStatus.INVOICED)`  vu qu'il n'y a pas d'autres paramètres ?

Reply:
C'est ce que je faisais dans les tests mais je trouvais que ca faisait beaucoup de code pour pas grand chose :thinking: A la limite si faire une nouvelle factory te gêne on peut créer une fixture directement dans le fichier de test j'y avais pas pensé